docker-compose.yml

```diff
@@ -10,10 +10,12 @@ services:
     build:
       context: .
       dockerfile: ./test/Dockerfile
-    command: bash -c 'while !</dev/tcp/domain_db/5432; do sleep 1; done; just serve-air'
+    command: just serve-air
     depends_on:
-      - domain_db
-      - redis
+      postgres_db:
+        condition: service_healthy
```

Comment:
I'm pretty surprised to see this.

Reply:
Yeah I've been hoping for a better way to handle this. It relies on the `healthcheck` config set on the service it's depending on.